### ISSUE TITLE
support for suppressing "SELECT N" trailing lines

### ISSUE THREAD

**Issue by teepark** [dbcli/pgcli#1374]
## Description
When piping output from pgcli, or using a pager such as VisiData, the trailing line after read query results like "SELECT 14" is more of an annoyance than anything else. There should be some means of suppressing it.

- it could be part of the definition of `less_chatty`
- it could be automatically suppressed when `table_format = csv`
- there could be a new configuration setting for it

## Your environment
Mac OS 12.5.1
pgcli 3.5.0
```
$ pip freeze
protobuf==4.21.8
```

- [X] Please provide your OS and version information.
- [X] Please provide your CLI version.
- [X] What is the output of ``pip freeze`` command.

**Comment by j-bennet** [comment]
> it could be automatically suppressed when `table_format = csv`

We did this for a long time. We only started displaying status in CSV with this PR, merged on Sep 2 2022:

https://github.com/dbcli/pgcli/pull/1341

The PR fixed an issue with visidata pager, where it would swallow stderr, and in case an error happened during query, you'd see nothing:

https://github.com/dbcli/pgcli/issues/1326.

The output like `SELECT 14` may be not very useful, but it doesn't break anything, and in case of an error, you do want to see error output.

I don't think we need to fix the current behavior.

**Comment by teepark** [comment]
Maybe the disconnect here is that kbd in #1326 was getting those additional status lines sent to stderr?

In my case with pgcli 3.5.0 those lines are getting thrown into stdout along with the actual CSV (which [appears intentional](https://github.com/dbcli/pgcli/blob/main/pgcli/main.py#L1612)), and it does very much break things:
![tmuxin_Code_backend_and_Hacker_News](https://user-images.githubusercontent.com/37182/201416370-de591ad8-f520-4549-9d72-23271deb464f.png)

Is there a setting I'm missing that directs trailing status lines to stderr?

**Comment by teepark** [comment]
I think I get what happened on #1326 - litecli sends trailing status lines to stderr, so @kbd assumed pgcli was doing the same and phrased the issue as "why aren't I getting stderr?". @kbd actually never confirmed their problem was solved on that thread either, I assume they're seeing the same thing I am now.

Litecli's approach really makes the most sense. With pgcli only using stdout the choices are between including status lines that might look right in the terminal but will break programmatic consumers expecting just CSV, or omitting them entirely and having the user potentially miss out on important error messages. Sending status and error lines to stderr gives the best of both worlds - the stdout that's piped into visidata or other CSV-consuming downstream pager programs is all true CSV, meanwhile the user also has the opportunity to see errors in the terminal.

If you're amenable I'd like to either reopen this and update the issue title or create a new separate issue.

---

FWIW I can hack around the problem by using this script as the pager:
```sh
( sed -n -e '/^"/p' -e '/^[^"]/q'; cat >&2 ) | vd -f csv
```
but this is pretty brittle, for instance relying on the fact that pgcli always outputs CSV with every field "double-quoted", and that status/error lines never begin with double quotes.

**Comment by teepark** [comment]
I've resolved this in #1383.
